src/app/Main/Body/QueryPage/Services/TableServices/calculateRowColSums.service.ts

```diff
@@ -4,7 +4,8 @@ function calcSums(
     tableData: Record<string, string>[],
     columnNames: string[],
     mapRowSums: (value: number, index: number) => string,
-    mapColSums: (value: Record<string, number>, key: string) => string
+    mapColSums: (value: Record<string, number>, key: string) => string,
+    mapTotalSum: () => string

```

Comment:
Haha. I was probably too enthusiastic about mapper. 
Done.

Reply:
Yes, it would have made sense if you had given the actual calculated total sum to the mapper to let the caller decide what to do with it, that would be in the actual code just ignoring it. If the table sum algorithm would be an exported service it would have made more sense to use an actual mapper.